docs/content/latest/api/ysql/the-sql-language/statements/ddl_create_index.md

```diff
@@ -47,9 +47,57 @@ Use the `CREATE INDEX` statement to create an index on the specified columns of
 
 `CONCURRENTLY`, `USING method`, `COLLATE`, and `TABLESPACE` options are not yet supported.
 
+When an index is created on a populated table, YugabyteDB automatically
```

Comment:
Overall comment: please don't break lines in markdown.

There are two good options in markdown:
* one line per paragraph, as you would with a word processor
* one line per sentence — if we all did this, diffs would be easier to parse, but we don't...